### PR TITLE
Egg timer code nit & corrections

### PR DESCRIPTION
**Pull request by Fang-**
This triggered me. Also we were using `card` in place of `move` a bunch? Pretty sure the code wouldn't compile.

I have a couple other style changes I'd make, but they're a bit more aggressive, so just listing them here instead:
- `(pair bone card)` -> `[bone card]` because you never ever care about accessing these and `+pair` isn't exactly modern hoon.
- `+>.$` -> `+*  this  .` etc.
- `:_  ~  move` ->  `[move]~`, double-`:_`s are pretty hard to follow imo, and `[item]~` lists are pretty common in these contexts. (Alternatively, `[move ~]`.)

### CODE REVIEW

File: tutorials/hoon/egg-timer.md [hunk 1]
```diff
@@ -96,12 +96,12 @@ The final part of this `move` is:
 
 `now` is the current time of type `@da`, and `t` was declared as `@dr`. Because they are both atoms, we can add `now` and `t` these two to get an atom that is `t` units of time into the future from `now`. That produced atom can be interpreted as a `@da`.
 
-That's all for our `poke-noun` arm. But what about when the timer goes off? Behn will create a `gift`, a similar construct to how we created an `card`, only this time it will end up being dispatched back to us via Gall in the `++wake` arm. Any app that wants to use a timer trigger needs to have an arm called `++wake`.
+That's all for our `poke-noun` arm. But what about when the timer goes off? Behn will create a `gift`, a similar construct to how we created a `card`, only this time it will end up being dispatched back to us via Gall in the `++wake` arm. Any app that wants to use a timer trigger needs to have an arm called `++wake`.
```
Comment (ohAitch) [inline tutorials/hoon/egg-timer.md]: move?

Comment (Fang-) [reply tutorials/hoon/egg-timer.md]: I left this uncorrected because a `move` is just a `card` on a """path""" (`bone`), ie it's the effect itself, which is also true for `gift`.

File: tutorials/hoon/egg-timer.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ The first thing to notice is that we are creating a `core` (`|%`) and a `door` (
 
 ```hoon
 |%
-+$  card   (pair bone move)
++$  card  (pair bone move)
 +$  move  [%wait path @da]
```
Comment (ohAitch) [inline tutorials/hoon/egg-timer.md]: Okay right this is just backwards and I think the source of my confusion

Comment (Fang-) [reply tutorials/hoon/egg-timer.md]: Oh god I touched this and didn't even notice. Hah, thanks for the heads up!

Comment (ohAitch) [reply tutorials/hoon/egg-timer.md]: At least that answers the question of which parts of the code would and wouldn't compile!

**Comment by ohAitch** [comment]
Uh to reiterate, this branch currently contains the text "An `card` is a `pair` of `bone` and `move`" in contradiction with the corrected code, and did not seem ready for merge.
